### PR TITLE
Rate limiting

### PR DESCRIPTION
**Pull request by rcoleworld**
Rate limits the GET /articles request after 5 requests in 30 seconds.

### CODE REVIEW

File: client/src/Home.js [hunk 1]
```diff
@@ -93,20 +93,22 @@ const Home = () => {
   const [articles, setArticles] = useState([]);
   const [mostBiasArticles, setMostBiasArticles] = useState([]);
   const [leastBiasArticles, setLeastBiasArticles] = useState([]);
+  const [rateLimited, setRateLimited] = useState(false);
   // Recent Articles 
   useEffect(() => {
     axios
       .get(
-        "http://home.flores.sh:5001/articles?limit=18&orderBy=firstPublishDate&orderType=des"
+        "http://localhost:5001/articles?limit=18&orderBy=firstPublishDate&orderType=des"
```
Comment (sydneyTanderson21) [inline client/src/Home.js:101]: Is this supposed to be localhost?

**Review by sydneyTanderson21 — CHANGES_REQUESTED** [review]
Double-check to make sure you meant to change the URL from Marco's server to 'localhost'

**Comment by nxtHiro** [comment]
+1 to address double check

File: client/src/Home.js (hunk 1 above)
Comment (rcoleworld) [reply client/src/Home.js:101]: This should be fine since we are moving to .env variables anyway.

**Pull request merged**